activiti-spring-boot/spring-boot-starters/activiti-spring-boot-starter-hal-rest-api/src/test/java/org/activiti/Application.java

```diff
@@ -0,0 +1,40 @@
+package org.activiti;
+
+import org.activiti.engine.IdentityService;
+import org.activiti.engine.identity.Group;
+import org.activiti.engine.identity.User;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class Application {
+
+	public static void main(String[] args) {
+		SpringApplication.run(Application.class, args);
+	}
+
+    @Bean
+    InitializingBean usersAndGroupsInitializer(final IdentityService identityService) {
```

Comment:
I removed it for the moment, but we might get blocked when writing task related tests.